controller/pkg/secrets/cert.go

```diff
@@ -38,6 +36,12 @@ import (
 	operatorv1beta1 "github.com/kong/kubernetes-configuration/api/gateway-operator/v1beta1"
 )
 
+// SetCALogger sets the logger for the CFSSL signer. Call it once at the start
+// of the program to ensure that CFSSL logs are captured by the operator's logger.
+func SetCALogger(logger logr.Logger) {
+	cflog.SetLogger(loggerShim{logger: logger})
```

Comment:
Wouldn't it be better to retain the `sync.Once` wrapper here to prevent these issues in the future?

I agree on moving its call to `main` but just to be on the safe side and prevent troubles in the future I'd re-add the `sync.Once`. WDYT?